### DISCUSSION TITLE
Missing test suite?

### DISCUSSION THREAD

**Discussion by deeplook** [flet-dev/flet#2489] [General]
Stumbling over https://github.com/flet-dev/flet/discussions/2383 I was just looking for a test suite for flet and to my surprise I could not find any. There are also no GitHub actions for building and testing this package. I'm curious for the reasoning behind this. Surely there must be some testing for this package, isn't it? Why not use the powerful GitHub actions? There seems to be even a [macOS runner](https://github.blog/changelog/2022-06-13-github-actions-macos-12-for-github-hosted-runners-is-now-generally-available/).

I would love to see build tests along the lines of this code snippet (using brew on macOS, but apt-get will do it on Linux):

```bash
brew install --cask android-platform-tools
brew install --cask flutter
brew install openjdk
brew install gradle

flet create --template minimal minimal
flet build apk minimal
```

**Comment by frchalaoux** [comment]
Yes, I'm agree  stumbling too over https://github.com/flet-dev/flet/discussions/2383 I would also love to see build tests along the lines of this code snippet as deeplook said it.  Cheers, François-Régis
